src/test/steps/migrate_manifest.test.ts

```diff
@@ -2,65 +2,68 @@ import { expect } from "chai";
 import { merge, get } from "lodash";
 import * as memFs from "mem-fs";
 import * as fsEditor from "mem-fs-editor";
+import { mkdir, rm } from "shelljs";
 import subject from "../../steps/migrate_manifest";
 import { Map } from "immutable";
 
 describe("migrate manifest", () => {
   let editor;
   let options: Map<string, any>;
   const cwd = process.cwd();
-  const src = `${cwd}/src/test/fixtures/basic_ticket_sample_app`;
-  const dest = `${cwd}/tmp/test/basic_ticket_sample_app`;
+  const src = `${cwd}/tmp/test/v1_app`;
+  const dest = `${cwd}/tmp/test/v2_app`;
+  const srcManifestPath = `${src}/manifest.json`;
+  const destManifestPath = `${dest}/dist/manifest.json`;
 
   beforeEach(() => {
+    mkdir("-p", src);
+    mkdir("-p", dest);
     editor = fsEditor.create(memFs.create());
     options = Map({ src, dest, editor });
+    editor.writeJSON(srcManifestPath, {});
+  });
+
+  afterEach(() => {
+    rm("-rf", src);
+    rm("-rf", dest);
```

Comment:
#nit Should actually be able to consolidate this with the line above, i.e. https://github.com/shelljs/shelljs#rmoptions-file_array